$NetBSD: patch-ad,v 1.4 2011/08/10 12:01:13 adam Exp $

Perl in pkgsrc doesn't install or tweak perllocal.pod, so comment
out that test.

Also apply fix to test culled by following references from
http://rt.cpan.org/Public/Bug/Display.html?id=40698
Expected output is now only present when VERBINST=1.

--- t/INSTALL_BASE.t.orig	2009-02-20 00:30:32.000000000 +0000
+++ t/INSTALL_BASE.t	2009-04-12 00:49:30.000000000 +0000
@@ -10,7 +10,7 @@
 use File::Path;
 use Config;
 
-use Test::More tests => 20;
+use Test::More tests => 18;
 use MakeMaker::Test::Utils;
 use MakeMaker::Test::Setup::BFD;
 
@@ -42,7 +42,7 @@
 
 my $make = make_run();
 run("$make");   # this is necessary due to a dmake bug.
-my $install_out = run("$make install");
+my $install_out = run("$make install VERBINST=1");
 is( $?, 0, '  make install exited normally' ) || diag $install_out;
 like( $install_out, qr/^Installing /m );
 
@@ -52,7 +52,7 @@
   ('../dummy-install/lib/perl5/Big/Dummy.pm',
    '../dummy-install/lib/perl5/Big/Liar.pm',
    '../dummy-install/bin/program',
-   "../dummy-install/lib/perl5/$Config{archname}/perllocal.pod",
+#  "../dummy-install/lib/perl5/$Config{archname}/perllocal.pod",
    "../dummy-install/lib/perl5/$Config{archname}/auto/Big/Dummy/.packlist"
   );
 
